cmd/print.go

```diff
@@ -33,7 +33,7 @@ import (
 	// Call init function for the providers
 	_ "github.com/kubernetes-sigs/ingress2gateway/pkg/i2gw/providers/apisix"
 	_ "github.com/kubernetes-sigs/ingress2gateway/pkg/i2gw/providers/cilium"
-	_ "github.com/kubernetes-sigs/ingress2gateway/pkg/i2gw/providers/gce"
+	// _ "github.com/kubernetes-sigs/ingress2gateway/pkg/i2gw/providers/gce"
```

Comment:
Why is GCE singled out here?

Reply:
I talk about it in the PR comments, but GCE both reads GCE CRDs and outputs GCE CRDs so it didn't fit in quite as well into the provider/emitter architecture. I'll add it back, was just curious for some thoughts.